src/ascent/runtimes/ascent_main_runtime.cpp

```diff
@@ -1392,7 +1392,12 @@ AscentRuntime::FindRenders(conduit::Node &image_params,
 void
 AscentRuntime::BuildGraph(const conduit::Node &actions)
 {
-  // exection will be enforced in the following order:
+  // make sure to clean helpers that we use
+  // to build the graph
+  m_connections.reset();
+  m_scene_connections.reset();
+
```

Comment:
@mclarsen  -- are their other graph building related thing we need to clear here?
These look like the only two to me, but maybe you can give it a quick look?